### PR TITLE
Angular basic setup

### PR DESCRIPTION
**Pull request by devmanhinton**
@golivera 

NOT TO MERGE!

Just spent 30-40 min. on a badly documented feature of a grunt-plugin grrrrrrr

Anywho - still much more work to be done but this has:

- Auto loading of partials / custom angular js to `static/processed/`
- Hooking into our page of the app

Note that I just copied over the tutorial code and will slow shift it to match our app :smile_cat: 

I am going to grab :meat_on_bone: and maybe :coffee: and then :computer: or :zzz:

### CODE REVIEW

**Comment by devmanhinton** [comment]
Notice the P(Coffee) & P(work) as well as P(Coffee) * P(sleep) are not independent probabilities so you can't just multiple them out!!

**Comment by golivera** [comment]
Haha this guy! Thanks for getting this done :+1:

File: classpath/static/js/angular-js/app.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/* App Module */
+
+var phonecatApp = angular.module('phonecatApp', [
```
Comment (golivera) [inline classpath/static/js/angular-js/app.js]: Straight from the tutorial! :joy_cat:

Comment (devmanhinton) [inline classpath/static/js/angular-js/app.js]: haha Yeah, shifting it over should be easy. I love the lolcat hahah

File: classpath/Gruntfile.js [hunk 2]
```diff
@@ -11,10 +13,23 @@ module.exports = function(grunt) {
...
+            custom: {
+                src: [STATIC_ROOT + 'js/angular-js/*'],
+                dest: PROCESSED_ROOT + 'js/angular-app.js'
             }
         },
 
+        copy: {
+            main: {
+                src: '*',
+                expand: true,
```
Comment (devmanhinton) [inline classpath/Gruntfile.js]: Undocumented but `expand:true` is necessary to make `cwd: ...` work below. @golivera Does this merit a comment in the code?

Comment (golivera) [inline classpath/Gruntfile.js]: It never hurts to comment the code

**Pull request merged**